### ISSUE TITLE
failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_ADDRESS

### ISSUE THREAD

**Issue by mahaishou** [tensorflow/tensorflow#22492]
when I train keras_retinanet model with tf-gpu, the following errors occured. CUDA_ERROR_ILLEGAL_ADDRESS.
It's stable reproduction.

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
CentOS Linux release 7.4.1708
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
no
- **TensorFlow installed from (source or binary)**:
binary(pip install tensorflow-gpu)
- **TensorFlow version (use command below)**:
1.10.0-dev20180720
keras version = 2.2.0
- **Python version**:
Python 2.7.5
- **Bazel version (if compiling from source)**:
n/a
- **GCC/Compiler version (if compiling from source)**:
n/a
- **CUDA/cuDNN version**:
cuda version = 9.0.176
cudnn version = 7.1.4
- **GPU model and memory**:
Tesla P40 ,4 gpu , 22919MiB memory, Driver Version: 390.77
- **Exact command to reproduce**:
download open image datasets
git clone https://github.com/fizyr/keras-retinanet.git
pip install . --user
python setup.py build_ext --inplace
python keras_retinanet/bin/train.py --batch-size 16 --gpu 0,1,2,3 --multi-gpu 4 --multi-gpu-force  --steps 100 --snapshot-path oid_snapshots oid /var/oid --parent-label Person

Epoch 1/50
2018-09-25 14:57:31.436888: E tensorflow/stream_executor/cuda/cuda_driver.cc:1078] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.436888: E tensorflow/stream_executor/cuda/cuda_driver.cc:1078] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.436888: E tensorflow/stream_executor/cuda/cuda_driver.cc:1078] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.436893: E tensorflow/stream_executor/cuda/cuda_driver.cc:1078] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.436980: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 0x8365350: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437097: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 0x89fa330: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437068: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 0x7cd6b60: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437011: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 0x7622e90: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437161: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 0x7cd6b60: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437173: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 0x7622e90: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437113: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 0x8365350: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437123: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 0x89fa330: CUDA_ERROR_ILLEGAL_ADDRESS
2018-09-25 14:57:31.437207: F tensorflow/stream_executor/cuda/cuda_dnn.cc:211] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.
2018-09-25 14:57:31.437211: F tensorflow/stream_executor/cuda/cuda_dnn.cc:211] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.
2018-09-25 14:57:31.437315: F tensorflow/stream_executor/cuda/cuda_dnn.cc:211] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.
2018-09-25 14:57:31.437303: F tensorflow/stream_executor/cuda/cuda_dnn.cc:211] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

code address:https://github.com/fizyr/keras-retinanet

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by wt-huang** [comment]
@mahaishou Could you install CUDA9.2? Also, use `pip install tf-nightly` to install TensorFlow.

**Comment by mahaishou** [comment]
@wt-huang ok, I will try this and update this issue later.

**Comment by wt-huang** [comment]
Is this resolved?
